sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -1519,6 +1610,12 @@ public partial class MetricDimension
         public string Operator { get { throw null; } set { } }
         public System.Collections.Generic.IList<string> Values { get { throw null; } }
     }
+    public partial class Metrics
```

Comment:
Single word is not allowed, so please use rename-mapping to rename it with a more describe name

Reply:
Renamed to "MetricProperties"

Reply:
Well still this name is a bit generic. Could we add some more words to specify this class?
The arch board has a requirement for dotnet SDKs - no class could share the same name even if they come from different SDK with different namespaces.
This name looks generic and has high possibility to collide with names in other RPs.

Reply:
Renamed to "MonitorWorkspaceMetricProperties"